data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/model/TransformerValueDeserializer.java

```diff
@@ -0,0 +1,33 @@
+package org.opensearch.dataprepper.pipeline.parser.model;
+
+import com.fasterxml.jackson.core.JsonParser;
+        import com.fasterxml.jackson.databind.DeserializationContext;
+        import com.fasterxml.jackson.databind.ObjectMapper;
+        import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+        import java.io.IOException;
+        import java.util.Map;
+
+public class TransformerValueDeserializer extends StdDeserializer<Object> {
```

Comment:
I don't see this in use. Can we remove it?

Reply:
removed the file. addressed in the latest pr